terraform/xmtp-validation-service/_variables.tf

```diff
@@ -0,0 +1,54 @@
+variable "vpc_id" {
+  description = "The VPC ID"
+}
+
+variable "cluster_id" {
+  description = "The ID of the cluster to deploy into"
+}
+
+variable "private_subnets" {
```

Comment:
can you give this a type? I think its `list(int)`. Its not obvious how to set this without that check.